src/main/java/dk/kb/util/YAML.java

```diff
@@ -274,6 +273,48 @@ public static YAML resolveConfig(String configName) throws IOException {
         return resolveConfig(configName, null);
     }
 
+    /**
+     * Parse the given configStream as YAML.
+     * @param yamlStream YAML.
+     * @return a YAML based on the given stream.
+     */
+    public static YAML parse(InputStream yamlStream) {
+        Object raw = new Yaml().load(yamlStream);
+        if (!(raw instanceof Map)) {
+            throw new IllegalArgumentException("The config resource does not evaluate to a valid YAML configuration.");
+        }
+        YAML rootMap = new YAML((Map<String, Object>) raw);
+        log.trace("Parsed YAML config stream");
+        return rootMap;
+    }
+
+    /**
+     * Resolve the given YAML configurations and present a merged YAML from that.
+     * Note: This method merges the YAML configs as-is, which means that they cannot have identical keys at
+     * the root level.
```

Comment:
What happens if they do have root level key collitions? Does it result in an error/exception, or is it the last one to set a key that wins?

Reply:
Fixed by latest commit #66bc3cc1998f6d9f14e944bdddc4f760bc80bcc8

Reply:
(the latest one wins)